src/EventStore.Core.Tests/Http/HttpBehaviorSpecification.cs

```diff
@@ -31,7 +31,7 @@ public abstract class HttpBehaviorSpecification : SpecificationWithDirectoryPerT
         protected string _lastResponseBody;
         protected byte[] _lastResponseBytes;
         protected JsonException _lastJsonException;
-//MONOCHECK Does this work now?
```

Comment:
This file suffered the same fate as the other one mentioned above about being just reformatted. Would you mind terribly to just revert this particular file?